### ISSUE TITLE
[Flagship plugin] My Jetpack: some cards have nested interactive elements (CRM & Boost)

### ISSUE THREAD

**Issue by monsieur-z** [Automattic/jetpack#31619]
p1686670108551479-slack-C0D96691V

**Comment by andrii-lysenko** [comment]
Done here: https://github.com/Automattic/jetpack/pull/31611
